src/api/src/services/BinaryFlashingStrategy/AmazonS3/index.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Can we move `AmazonS3` out of `BinaryFlashingStrategy`? It appears to be loosely related to `BinaryFlashingStrategy` and looks nice as a standalone lib.